cpp/inc/bond/ext/grpc/detail/service.h

```diff
@@ -34,8 +34,29 @@
 
 namespace bond { namespace ext { namespace gRPC
 {
+    class server;
 
-class server;
+    namespace detail
+    {
+        class service;
+
+    } // namespace detail
+
+
+    /// @brief Base public class that all Bond gRPC++ services inherit.
```

Comment:
Should this be moved to service.h or some other public header?

Reply:
Will move to `abstract_service.h`.